docs/docsite/rst/dev_guide/developing_program_flow_modules.rst

```diff
@@ -517,7 +517,7 @@ Why pass args over stdin?
 
 Passing arguments via stdin was chosen for the following reasons:
 
-* When combined with :ref:`pipelining`, this keeps the module's arguments from
+* When combined with :ref:`flow_pipelining`, this keeps the module's arguments from
```

Comment:
I think this one should point to user docs (or perhaps flow_pipelining should point to user docs) but I'm not sure where the canonical location for those docs is.  There's a section on pipelining in ./user_guide/intro_inventory.rst and the generated ./reference_appendices/config.rst also has information on pipelining.

Reply:
I pointed it to the Config page in reference_appendices for now.